### DISCUSSION TITLE
cloud-native/kubernetes/k8s-release-strategy

### DISCUSSION THREAD

**Discussion by giscus** [GuoXiCheng/guoxicheng.github.io#191] [Announcements]
# cloud-native/kubernetes/k8s-release-strategy

K8S应用发布策略 重建部署 定义：重建部署是最简单的发布策略 部署方式：先停止并删除旧版本容器，再创建并启动新版本容器，部署中会导致应用程序短暂不可用 适用场景：无需保证应用持续可用，例如开发环境 滚动更新 定义：滚动更新是一种逐步替换旧容器的策略 部署方式：逐步将新版本的容器添加到集群中，同时逐步删除旧版本的容器 适用场景：确保应用程序持续可用 蓝...

https://guoxicheng.top/cloud-native/kubernetes/k8s-release-strategy.html

**Comment by Paper-Dragon** [comment]
大佬写的超级好，很少有人图形化表示这个过程，这个文章很形象了！
